### ISSUE TITLE
Access previous frame fragment shader

### ISSUE THREAD

**Issue by Guilouf** [glumpy/glumpy#236]
I am playing with the examples, specially framebuffer,  i know how to pass a shader as a texture to another shader using framebuffer, but how to pass to a shader its previous state, like in shadertoy ? Could'nt find any example like this

**Comment by rougier** [comment]
What do you mean by previous state exactly? Do you have a (simple) shadertoy example?

**Comment by Guilouf** [comment]
https://www.shadertoy.com/view/tldGzf I made this one as the simplest example. The buffer A takes itself as input. This can be done in OpenGL when you don't clear the frame buffer.

**Comment by rougier** [comment]
What is the state you pass exactly (i.e. which variable)?

**Comment by Guilouf** [comment]
Hum im not sure to understand the question, the state i refer is the shader rendered as a texture, i.e `iChannel0` in the bufferA of the shadertoy example. In fact its rgba values of the previous render of the buffer.

**Comment by rougier** [comment]
Oh yes, sorry. Did you tried playing with the [grayscott](https://github.com/glumpy/glumpy/blob/master/examples/grayscott.py) example?

**Comment by Guilouf** [comment]
Yes i did, but i didn't dig in as it only display some kind of blurred cross, wich may not be the intended result. (i have a warning ` Backend (<module 'glumpy.app.window.backends.backend_glfw' not available`. I will try to look into more precisely

**Comment by rougier** [comment]
It's a cellular automata simulation where the computation is made on the GPU. It's taking advantage of two framebuffers (one for reading state, one for writing the new one). Result should be something like https://www.labri.fr/perso/nrougier/from-python-to-numpy/data/gray-scott-1.mp4

**Comment by Guilouf** [comment]
Well for me after one second it stays like this: 
![image](https://user-images.githubusercontent.com/10614714/71578773-d3642380-2af9-11ea-923d-e4a085775302.png)
Anyway the process you describe i need

**Comment by rougier** [comment]
Do you get the same with other sets of parameters ?

**Comment by Guilouf** [comment]
When i play with parameters of UV sometimes i get a square instead of a cross.

**Comment by rougier** [comment]
Hmmm. Does it evolve a bit or does it start straight as square/cross?

**Comment by Guilouf** [comment]
It starts with a full square and then rapidly evolves to a cross

**Comment by rougier** [comment]
Ok, so something is computed. This means you can probably use the script to do a simpler computation like a diffusion process (color = max of four neighbors) and start with a single lit pixel at the center.
